db/migrate/20200213150314_create_service_credential_types.rb

```diff
@@ -0,0 +1,30 @@
+class CreateServiceCredentialTypes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :service_credential_types do |t|
+      t.string :name
+      t.string :description
+      t.string :kind
+      t.string :namespace
+
+      t.string :source_ref, :null => false
+
+      t.datetime :resource_timestamp
+      t.jsonb :resource_timestamps, :default => {}
+      t.datetime :resource_timestamps_max
+
+      t.timestamps
+      t.datetime :archived_at
+      t.datetime :source_created_at
+      t.datetime :source_updated_at
```

Comment:
> We don't have source_updated_at for other models, so I'm for removit it from here too.

[We collect `source_updated_at` for `ServiceOfferingNode` as  Ansible Tower API provides column `modified`](https://github.com/RedHatInsights/topological_inventory-ansible_tower/blob/7176eea240a9926d1720f74590e2b4ecce949513/lib/topological_inventory/ansible_tower/parser/service_offering_node.rb#L13)

And there is `source_updated_at` on `Credentials` - based on your note -  we should consider removal of `source_updated_at ` in `Credentials ` and `ServiceOfferingNode`. Otherwise there are no other `source_updated_at` column as you pointed out.

Anyway ok I will remove it 👍